compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/distribute_l1_allocations.mlir

```diff
@@ -90,4 +90,27 @@ func.func @non_distributing_subview(%index : index) {
 }
 
 
+// -----
+
+// Example where the subview type is a complete view of the alloc: unchanged IR
+
+// CHECK-LABEL: @complete_view_subview
+// CHECK: %[[ALLOC:.+]] = memref.alloc() : memref<4xbf16, 2>
+// CHECK: scf.forall (%[[ARG0:.+]]) in (4) {
+// CHECK: %[[C0_BF16:.+]] = arith.constant
+// CHECK: %[[SUBVIEW:.+]] = memref.subview
```

Comment:
I think you can remove these CHECKS?

Reply:
I wanted to check that the IR was unchanged (i.e. no new created). Is there a problem with including these CHECKS? I changed them to to CHECK-NEXTs to be absolutely sure no new allocs are created.

Reply:
I was thinking checking the alloc and subview shapes are enough to proof the IR is unchanged. There seems no need to check scf.for ops and arith.constant/

Reply:
Okay, I don't want to block on this, but I think it's better to check the alloc shape as what you did in the first version. You can use CHECK-NEXT as is in between.

```
// CHECK: %[[ALLOC:.+]] = memref.alloc() : memref<4xbf16, 2>
...
// CHECK-SAME: %[[ALLOC]][0] [4] [1] : memref<4xbf16, 2> to memref<4xbf16, 2>
```